### DISCUSSION TITLE
is the project still active?

### DISCUSSION THREAD

**Discussion by sonht-0555** [thenick775/gbajs3#155] [Q&A] [answered]
@thenick775 I haven't seen you active on the gbajs branch lately, are you busy with work or are you preparing for a new NBA core? 🙌 Sorry for being curious, I'm excitedly waiting for your new core 😂

**Comment by thenick775** [comment, marked as answer]
Yeah! I'm actually going through a job change right now, while also focusing on some of my go repos and other js repos like the pwa prompt!

So my attention has moved for a month or two, but rest assured I'm still working on this project! 😎 

I'm currently working through threading in the mGBA core, since to use NBA I will need to use threads, or find a way to get around COOP/COEP headers (not using threads for NBA).

Both of these should get rolled out in what I estimate will be ~2-3 months or so 😎

**Comment by sonht-0555** [comment]
@thenick775 Ya, it's worth looking forward to, I'll be waiting to test this combination.  I thought you left it 😋

**Comment by thenick775** [comment]
I've created issue #156 to cover the addition of another core (or two) 😎

I have no real ETA, but this is now concretely anchored in the roadmap!🚀

**Comment by sonht-0555** [comment]
@thenick775 It's worth looking forward to.  When there is a `wasm` and `pre.js` version of one of the two cores above, please let me know.  I will try to attach it to the UI to test the functionality with you.
